src/components/navbar/Navbar.tsx

```diff
@@ -7,7 +7,7 @@ import ListItem from "@mui/material/ListItem";
 import ListItemButton from "@mui/material/ListItemButton";
 import ListItemIcon from "@mui/material/ListItemIcon";
 import ListItemText from "@mui/material/ListItemText";
-import { mainNavbarItems } from "./NavbarListItems";
+import { mainNavbarItems } from "./navbarListItems";
```

Comment:
Should be named with a big N this is why we have an error

Reply:
I really don't know why this keeps on failing...